src/doc/gui_manual/Operators/OperatorTypes/CreateBonds_operator.rst

```diff
@@ -0,0 +1,76 @@
+
+Note: 
+
+1. A "*" matches any species.
+
+2. It does not matter which species is *1st* and which is *2nd*. The bonds are not unidirectional. 
+
+3. The first match in the list is taken, even if later lines also match, which allows you to specify more specific rules above less-specific rules.
+
+For example, if the first line is "H", "*", "0.4", "1.2", this specifies that the algorithm should create a bond between two atoms if either one is Hydrogen and the distance between them is between 0.4 and 1.2 spatial units.
+
+As a follow-on to this example, suppose the second line is "*", "*", "0.4", and "1.9". Now suppose two atoms exist, H and O, and they are separated by a distance of 1.5 units. Because one is H, it will match the first line, determine the distance is too great (since it's greater than 1.2), and so it will not create a bond between this atom pair. Since this atom pair matched the first line, the second line is not considered, even though the atom pair matches its criteria.
+
+Just below the actual bonding rules list are several buttons: The **New** button creates a new rule, and **Del** deletes it.  **Up** moves the currently selected rule up in the list, and **Down** moves the currently selected rule later in the list. Recall that the order of rules matters because only the first match is considered. 
+
+The **Details** section contains controls to set the values for a rule.  The **1st** and **2nd** controls pop up the species selection widget shown in :numref:`Figure %s <createbonds-selector>`.
+
+.. _createbonds-selector:
+
+.. figure:: images/createbonds-selector.png
+
+
+To get a wildcard which will match any type of atom, choose **Match any element** at the bottom; it is selectable just as any individual species in the periodic table.
+
+Also, note that there is the possibility for some *hinting* to help guide your selection to the viable types of atoms. (This depends on conditions like file format support.) For example, in this screenshot, the **H** and **Si** elements are in boldface, since the file contains only those types of atoms.  The **Min** and **Max** fields are standard text widgets.  
+
+Advanced settings
+"""""""""""""""""
+
+.. _createbonds_advanced:
+
+.. figure:: images/createbonds_advanced.png
+
+
+**Variable for atomic number** defaults to *element* as per the convention, but can be set to any integral variable corresponding to the atomic number of each atom.
+
+If you specify the wrong distance, each atom might try to bond to many other atoms. To keep an error like this from causing a severe hit to memory and performance, **Maximum bonds per atom** will stop the process before it gets out of hand.  The default value is *10*, and could safely be set lower in many cases, but is user-settable for unusual cases where >10 bonds are needed on some atoms. 
```

Comment:
Grammar: "...value is 10, and could safely be set lower in many cases, but is user-settable..."
Both commas in this sentence can either be removed or altered like so:
"...value is 10, and it could safely be set lower in many cases, but it is user-settable..."